lib/rides/widgets/search_ride_filter.dart

```diff
@@ -300,9 +314,7 @@ class SearchRideFilter {
         if (_minComfortRating != _defaultRating ||
             _minSafetyRating != _defaultRating ||
             _minReliabilityRating != _defaultRating ||
-            _minHospitalityRating != _defaultRating) {
-          ratingWidgets.add(const SizedBox(width: 4));
```

Comment:
this was removed for what reason? Now the `if` is useless as far as i can see

Reply:
Haha! Yeah you're right and I removed the `if`. That was the tiny spacing issue I was talking about. Turns out, I counted the length of the `ratingWidgets` in order to loop over it when inserting `SizedBox`es between every category review. This line was supposed to add a little more spacing between the overall and category ratings. But turns out, it adds a ratingWidget I failed to account for and there was always one `SizedBox` too much at the end. Anyways, I realized how dumb that was and now I'm adding the extra "overallRating" `SizedBox` _after_ the other `SizedBox`es